### PR TITLE
Space Travellers' Hub

### PR DESCRIPTION
**Pull request by katarighe**
This is a collaborative work by @katarighe, @M-AminAlizadeh & @MargaretTechworld on the project Space Travellers' Hub. 

The **Space Travelers' Hub** is a single-page application that allows users to reserve rockets and dragons for missions. The missions are fetched from external APIs, and the selected services are displayed on the user's profile page.

Here are the general requirements met for this project:

- There are **[no linter errors](https://github.com/microverseinc/linters-config)**.
- Used **correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md)**.
- Documented my work **[in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)**.

The following project requirements have been accomplished successfully: 

- **Configured** & setup react and redux.
- **Fetched** data and updated Redux store for rockets, missions and dragons
- **Rendered** the data in the UI.
- **Wrote** actions and reducers for booking rockets + dragons and joining missions.
- **Wrote** actions and reducers for canceling rockets + dragons and leaving missions.
- **Used** conditional components rendering.
- **Created** My Profile section showing all of the 3 components. 
- **Completed** the technical setup of the project. 
- **Tested** all components with Jest and snapshots. 
- **Created** a README file describing the overall project. 

**Screenshots:**

image

image

image

image

### CODE REVIEW

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/test/Dragons.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (katarighe) [reply src/test/Dragons.test.js:1]: That is done, I have added all of it to my components and functions.

**Review by prolajumokeoni — APPROVED** [review]
Hi @katarighe 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- Tests added for all components  ✅ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by katarighe** [comment]
Thank you everyone! It is really appreciated. I would also like to thank my colleagues for their hard work! 🙏😊

**Pull request merged**